Nos/Views/NoteComposer/NoteComposer.swift

```diff
@@ -201,19 +228,47 @@ struct NoteComposer: View {
         }
         isPresented = false
     }
+    
+    private func loadQuotedNote() {
+        guard let quotedNoteID else {
+            return
+        }
+        
+        quotedNote = try? Event.findOrCreateStubBy(
+            id: quotedNoteID,
+            context: persistenceController.viewContext
+        )
+    }
 
+    private var canPost: Bool {
```

Comment:
I'm torn on the name of this. Above, I wish we could say something like `.disabled(postButtonDisabled)` so it doesn't have to have `!canPost`. Below, I like `guard canPost`. So I don't know if there's a compromise where we could call it something like `isPostEnabled` or if that's less clear. Above, I think I'd prefer to see `!isPostEnabled` over `!canPost`.

Or to be even more verbose: `isPostButtonEnabled`, maybe.

Reply:
I never like to state Bools negatively, so I'm slightly annoyed that Apple chose "disabled" for that view modifier name. But yeah, I think `isPostEnabled` is better, because it's not just the button that is enabled, but also posting at all.

Reply:
Done!